src/fingerprint.ts

```diff
@@ -78,6 +78,14 @@ export interface FingerprintData {
 
   // Must be sorted.
   dependencies: {[dependency: ScriptReferenceString]: FingerprintData};
+
+  service:
```

Comment:
No action required (or a comment could help). Why doesn't the boolean decision of whether a script is a service not need to be fingerprinted?

Reply:
Because services can never be cached and don't write out a fingerprint file, so there's no bad interaction really possible.

However, we do *now* store whether something is a service, since `service` will only be `undefined` if it is not a service at all.